EditPortfolio.html

```diff
@@ -0,0 +1,129 @@
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Fjalla+One&family=Francois+One&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body id="body_main">
+    <nav class="navbar">
+      <ul>
+        <li><a href="/index.html">Go Back To Home</a></li>
+      </ul>
+    </nav>
+
+    <div class="sidebar">
+      <div>
+        <ul>
+          <li>
+            <img
+              src="https://i.pinimg.com/originals/97/31/02/9731022f0be7c965e880505461643850.jpg"
+              alt="My Avatar"
+              class="avatar"
+            />
+          </li>
+          <li>
+            <div class="skillsContainer">
+              <h1 class="introduction_text">
+                Hi, I'm <span class="imgNameFont">Umar Ali</span>
+              </h1>
+              <h1 class="skillsHeading">Skills</h1>
+              <p class="skills">React</p>
+              <p class="skills">Bootstrap</p>
+              <p class="skills">.NET</p>
+              <p class="skills">Android</p>
+              <p class="skills">API</p>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <div id="container">
+      <section id="about" class="section">
+        <h1 class="title">Full-Stack Developer</h1>
+        <div class="border">
+          <h2 class="heading2">About</h2>
+          <p class="paragraph">
+            Dedicated and efficient full stack developer with 6+ years
+            experience in application layers, presentation layers, and
+            databases. Certified in both F/E and B/E technologies. Spearheaded
+            successful transition from LAMP stack to MEAN which cut latency by
+            40% and increased effectiveness of database administrators by 20%.
+            Seeking to further improve HTML5 and CSS3 skills as the future full
+            stack developer at Atmospheric Solutions.
+          </p>
+        </div>
+      </section>
+
+      <section id="experience" class="section">
+        <div class="border">
+
+
+          <div class="experiences">
+            <h2 class="heading2">Experiences</h2>
+            <ul id="experienceList">
+            </ul>
+          </div>
+          <button id="addExperienceBtn" class="addButton">Add Experience</button>
+      <div class="model">
+
+      </div>
+          <div id="experienceForm" class="model" >
+      
+          <div id="formContainer" class="formContainer">
+            <div class="model-content">
+              <h1 >Add Experience</h1>
+              <form id="experienceEntryForm">
+      
+                <label for="companyName">Company Name:</label>
+                <input type="text" id="companyName" required><br>
+          
```

Comment:
You should avoid using \<br> tag to give spaces between elements. It should be used for line breaks only. Here it is good practise to use css to give spacing.